package-lock.json

```diff
@@ -32,7 +33,7 @@
         "replace-in-file": "6.3.5",
         "rimraf": "3.0.2",
         "standard": "17.0.0",
-        "tap": "16.3.0",
+        "tap": "^16.3.0",
```

Comment:
hummm... `npm audit` tells '16.x' has moderate severity vulnerabilities . But when we run `npm audit fix --force`, `tap` is downgraded to '12.x` and it has high vulnerabilities. And once again, when we run `audit --fix` it's updagraded to `16.x` agian.  It never ends..

Reply:
<div><img src="https://media0.giphy.com/media/SXsLZPFD4wRqZqqI37/200.gif?cid=5a38a5a20jczzqy9tx1oxnbs8mpxzvg2zkjktolhz24l9jhl&amp;rid=200.gif&amp;ct=g" style="border:0;height:170px;width:300px"/><br/>via <a href="https://giphy.com/gifs/doctor-strange-SXsLZPFD4wRqZqqI37">GIPHY</a></div>

Reply:
It's gone!